web/modules/custom/joinup_core/tests/src/JoinupWorkflowTestBase.php

```diff
@@ -115,6 +127,14 @@ protected function createOgMembership(EntityInterface $group, AccountInterface $
   protected abstract function getEntityType();
 
   /**
+   * Returns the entity bundle for the tested node type.
+   *
+   * @return string
+   *    The entity bundle machine name.
+   */
+  abstract protected function getEntityBundle();
```

Comment:
This will be used later for the RDF test base. As discussed, we can leave it there for now.